parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/ZstandardCodec.java

```diff
@@ -91,7 +102,14 @@ public CompressionOutputStream createOutputStream(OutputStream stream, Compresso
 
   @Override
   public CompressionOutputStream createOutputStream(OutputStream stream) throws IOException {
-    return new ZstdCompressorStream(stream, conf.getInt(PARQUET_COMPRESS_ZSTD_LEVEL, DEFAULT_PARQUET_COMPRESS_ZSTD_LEVEL),
+    BufferPool pool;
```

Comment:
Thanks Dongjoon for working on this! 

It is kind of late. Just a minor comment: if you can wrap the code into a method and call it in both CompressionInputStream() and CompressionOutputStream, it would avoid duplicating. Not a big deal though.